src/auth/GetWindowsCert.ts

```diff
@@ -0,0 +1,29 @@
+import * as ca from "win-ca";
+import * as caApi from "win-ca/api";
+import * as crypt from "crypto";
+
+const useWindowsCertsThumbprint = (id: string) => {
+  const thumbprint = (cert: string) => {
+    var shasum = crypt.createHash("sha1");
+    shasum.update(Buffer.from(cert, "base64"));
+    return shasum.digest("hex").toUpperCase();
+  };
+
+  const list: string[] = [];
+
+  caApi({
+    store: ["My"],
```

Comment:
Should users be able to choose which store they want to use or is it always `Me`?

Reply:
Its always My, as this is a store in the current users store. Which the user will have access to without requiring elevated permissions.